projects/plugins/jetpack/modules/search/instant-search/components/dom-event-handler.jsx

```diff
@@ -169,6 +182,20 @@ export default class DomEventHandler extends Component {
 		}
 	};
 
+	scrollOverlayToTop = event => {
+		// NOTE: the propertyName need to be aligned with the animation
+		if ( event?.propertyName !== 'opacity' ) {
+			return;
+		}
+		// NOTE: IE11 doesn't support scrollTo. Manually set overlay element's scrollTop.
```

Comment:
We could potentially drop this (IE11 polyfill is being removed in https://github.com/Automattic/jetpack/pull/20499).

Reply:
removed in 0804f0234b